backend/alembic/versions/5e46d4ff6f21_change_type.py

```diff
@@ -0,0 +1,24 @@
+"""ChangeType for restoring deleted items
+
+Revision ID: 5e46d4ff6f21
+Revises: 8a3f36ffa8df
+Create Date: 2023-09-25 15:28:13.752183
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '5e46d4ff6f21'
+down_revision = '8a3f36ffa8df'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("ALTER TYPE changetype ADD VALUE IF NOT EXISTS 'RESTORE'")
+
+
+def downgrade():
+    pass
```

Comment:
If Postgres would allow it, yes. Unfortunately, removing values from an enum in Postgres is not supported :slightly_frowning_face: 

See: https://stackoverflow.com/a/25812436/1785740